### PR TITLE
chore(wallet): remove svelte wallet and make react default

### PR DESCRIPTION
**Pull request by samsiegart**
Tested with `agoric start --reset`, `agoric open`, and a basic user journey with the fungible faucet dapp. Also tested `yarn start` in `packages/dapp-wallet/ui`.

### CODE REVIEW

File: packages/solo/package.json [hunk 1]
```diff
@@ -30,7 +30,6 @@
     "@agoric/babel-standalone": "^7.14.3",
     "@agoric/captp": "^1.10.6",
     "@agoric/cosmic-swingset": "^0.34.2",
-    "@agoric/dapp-svelte-wallet": "^0.11.5",
```
Comment (michaelfig) [inline packages/solo/package.json:33]: This `package.json` needs to depend on `@agoric/dapp-wallet` instead.

File: packages/xsnap/package.json [hunk 2]
```diff
@@ -37,6 +37,7 @@
   },
   "devDependencies": {
     "@endo/base64": "^0.2.8",
+    "@rollup/plugin-commonjs": "^21.0.1",
```
Comment (michaelfig) [inline packages/xsnap/package.json:40]: Ah, good catch!

**Review by michaelfig — DISMISSED** [review]
LGTM!  Glad you got it working in both development and built mode.

File: packages/xsnap/package.json (hunk 2 above)
Comment (samsiegart) [reply packages/xsnap/package.json:40]: The sdk wouldn't build without this after removing the old wallet, not sure why /shrug

**Comment by dckc** [comment]
er... I suppose the name follows the `dapp-svelte-wallet` precedent, but is the wallet a dapp?

This discussion strongly suggests not:
> dapps have all of the domain specific knowledge, but none of the authority, and wallets have all of the authority and none of the domain specific knowledge.
-- https://github.com/Agoric/agoric-sdk/issues/3901#issuecomment-942649361

bikeshedding turned up the following names; I list them here in order of my preference:
  - `@agoric/powerbox` suggested by @michaelfig ; @samsiegart 's reaction reminds us that using ocap community jargon has a certain cost to it, though.
  -  `@agoric/web-wallet`
  - `@agoric/wallet`
  - `@agoric/offer-wallet`

File: packages/solo/package.json (hunk 1 above)
Comment (samsiegart) [reply packages/solo/package.json:33]: fixed

**Comment by michaelfig** [comment]
> er... I suppose the name follows the `dapp-svelte-wallet` precedent, but is the wallet a dapp?

> > dapps have all of the domain specific knowledge, but none of the authority, and wallets have all of the authority and none of the domain specific knowledge.

@samsiegart, can you make the move to just `packages/wallet/api` and `packages/wallet/ui`?  I'll flatten those in a later PR into `packages/wallet-api` and `packages/wallet-ui`.

**Comment by samsiegart** [comment]
> @samsiegart, can you make the move to just `packages/wallet/api` and `packages/wallet/ui`? I'll flatten those in a later PR into `packages/wallet-api` and `packages/wallet-ui`.

Done

**Review by michaelfig — APPROVED** [review]
BTW, also please adjust `lerna.json` to point to `wallet/` instead of `dapp-*/`.

**Pull request merged**